openssl-dynamic/src/main/c/tcn.h

```diff
@@ -103,6 +104,14 @@ jstring         tcn_new_stringn(JNIEnv *, const char *, size_t);
 
 #define AJP_TO_JSTRING(V)   (*e)->NewStringUTF((e), (V))
 
+#define TCN_CHECK_NULL(V, M, R)                      \
+    TCN_BEGIN_MACRO                                  \
+        if (V == NULL) {                             \
+            tcn_ThrowIllegalArgumentException(e, M); \
```

Comment:
@trustin @Scottmitch I am changing to a NPE but I think we still need `M` as the name of the field may be different (as its really the c field) compared to what the user provide us.